package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "My-To-do-list",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "start": "webpack-dev-server"
```

Comment:
- [ ] [OPTIONAL] The script option of your package.json file is not properly configured and hence your application does not launch in a browser when the command npm start is run. Kindly append this line below to the script options:

`start": "webpack serve --open`, in your packge.json file.